stdeb.cfg

```diff
@@ -0,0 +1,4 @@
+[DEFAULT]
+Depends = python-rdflib (>= 4.1.2), python-ply (>= 3.4), python-nose (>= 1.3)
+Copyright-File = LICENSE 
```

Comment:
Ah! great... I guess we will need a machine-readable copyright file in the future then.... But doss `stddeb` require its config file to be at the root? or should we move it to a `debian` sub-directory?

Also, would it be possible to have a minimal smoke test that would validate a package creation? 

NB: this reminds me that I dislike having a hard dependency on `nose` for tests. I created #15 tp track this.